src/main/java/net/coderbot/iris/Iris.java

```diff
@@ -229,6 +229,20 @@ private static void loadInternalShaderpack() {
 		internal = true;
 	}
 
+	public static boolean isValidShaderpack(Path pack) {
+		if (Files.isDirectory(pack)) {
+			return Files.exists(pack.resolve("shaders"));
+		}
+		if (pack.toString().endsWith(".zip")) {
+			try {
+				FileSystem zipSystem = FileSystems.newFileSystem(pack, Iris.class.getClassLoader());
+				return Files.exists(zipSystem.getPath("shaders"));
```

Comment:
Sometimes the shaders directory of the shaderpack is actually nested one directory, and Iris loads them irregardless, probably breaking the logic here.  See the comment [here](https://github.com/IrisShaders/Iris/pull/35#discussion_r549196794)

Reply:
You are correct, I will change the logic to match that of shader pack evaluation in the loading methods.